backend/s3/s3.go

```diff
@@ -842,6 +842,7 @@ const (
 	maxUploadParts      = 10000 // maximum allowed number of parts in a multi-part upload
 	minChunkSize        = fs.SizeSuffix(1024 * 1024 * 5)
 	defaultUploadCutoff = fs.SizeSuffix(200 * 1024 * 1024)
+	purgeBatchSize      = 1000 // the max number of objects allowed in a DeleteObjects request
```

Comment:
This number comes from AWS [document](https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html). 

> The request can contain a list of up to 1,000 keys that you want to delete.

It also works on Alibaba Cloud, but I'm kind of worried some uncommon S3 providers may set a lower limit on this. Should I set a lower value or add a new config option?